cypress/support/graphql-utils.ts

```
@@ -0,0 +1,147 @@
+
+/**
+ * Wait for a GraphQL operation to complete
+ * Use after aliasGraphQLOperation or mockGraphQLOperation
+ *
+ * @param operationName - The GraphQL operation name to wait for
+ * @returns Cypress chainable with the interception object
+ *
+ * @example
+ * cy.aliasGraphQLOperation('GetOrganizations');
+ * cy.visit('/orglist');
+ * cy.waitForGraphQLOperation('GetOrganizations').then((interception) => {
```

ESLint

[error] tsdoc-escape-greater-than: The ">" character should be escaped using a backslash to avoid confusion with an HTML tag

(tsdoc/syntax)